website/docs/r/infraleafs.html.markdown

```diff
@@ -1,18 +1,18 @@
 ---
 layout: "aci"
-page_title: "ACI: aci_switch_association"
+page_title: "ACI: aci_leaf_selector"
 sidebar_current: "docs-aci-resource-switch_association"
 description: |-
   Manages ACI Switch Association
 ---
```

Comment:
sidebar_current and description must be updated

Reply:
done, I have updated the PR